pkgs/development/compilers/zulu/default.nix

```diff
@@ -44,7 +49,7 @@ in stdenv.mkDerivation {
 
   pname = "zulu";
 
-  src = fetchurl {
+  src = if builtins.currentSystem == "aarch64-darwin" then aarch64-darwin-src else fetchurl {
```

Comment:
Please integrate this in to the previous logic and use stdenv.isX

Reply:
Done . Could you check it again